Kitodo/src/main/java/org/kitodo/production/services/command/AddDataScript.java

```diff
@@ -36,7 +36,7 @@ public void executeScript(LegacyMetsModsDigitalDocumentHelper metadataFile, Proc
 
         for (String value : metadataScript.getValues()) {
```

Comment:
`value` can occur at most once (see comment on `MetadataScript` class, replacement with `Optional<String>`). You can remove the loop.